locations/pipelines.py

```diff
@@ -75,6 +76,15 @@ def process_item(self, item, spider):
                 spider.crawler.stats.inc_value("atp/field/country/from_website_url")
                 item["country"] = country
 
+            # Still no country set, try an offline reverse geocoder.
+            lat, lon = item.get("lat"), item.get("lon")
+            if lat != None and lon != None:
```

Comment:
Can this be condensed to:

```suggestion
            if lat and lon:
```

(I've not actually checked this)

Reply:
That would exclude anything on the equator or the prime meridian.

Reply:
I think that's quite an edge case, but yeah lets not make a regression.